.env.test

```
@@ -0,0 +1,26 @@
+RATE_LIMIT_FALLBACK_MAX_REQUESTS=20
+RATE_LIMIT_CIRCUIT_BREAKER_THRESHOLD=5
+RATE_LIMIT_CIRCUIT_BREAKER_COOLDOWN_MS=60000
+
+# Stellar / USDC swap
+STELLAR_NETWORK=testnet
+STELLAR_HORIZON_URL=https://horizon-testnet.stellar.org
+STELLAR_SECRET_KEY=test-stellar-secret-key
+
+# Limits for retail, business, government (used by webhook suite)
+LIMITS_RETAIL=1000
+LIMITS_BUSINESS=5000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LIMITS_BUSINESS key should go before the LIMITS_RETAIL key

(UnorderedKey)